package.json

```diff
@@ -28,11 +28,11 @@
     "@types/node": "^13.13.4",
     "js-yaml": "^3.13.1",
     "typescript": "^3.8.3",
-    "vscode": "^1.1.37"
+    "@types/vscode": "^1.44.0"
   },
   "scripts": {
     "convert": "js-yaml syntaxes/mint.tmLanguage.yaml > syntaxes/mint.tmLanguage.json",
-    "postinstall": "node ./node_modules/vscode/bin/install && tsc"
```

Comment:
I got this line from https://github.com/jrieken/vscode-formatter-sample/blob/master/package.json#L37, but without the `vscode` dep, `node ./node_modules/vscode/bin/install` doesn't work. I'm not sure what it was for. Do we still need this?

Reply:
Do not need it, we can simply use typescript `tsc` to build the extension.

Reply:
This is also how Flutter does it, https://github.com/Dart-Code/Flutter/blob/master/package.json